scripts/get_localtime.py

```diff
@@ -0,0 +1,5 @@
+import time
```

Comment:
Add shebang line before import statement `#!/usr/bin/python3`
This way voicecommand can simply invoke your script without having to call python3